crates/sui-cluster-test/src/wallet_client.rs

```diff
@@ -24,9 +24,7 @@ impl WalletClient {
     pub async fn new_from_cluster(cluster: &(dyn Cluster + Sync + Send)) -> Self {
         let key = cluster.user_key();
         let address: SuiAddress = key.public().into();
-        let wallet_context = new_wallet_context_from_cluster(cluster, key)
```

Comment:
same here - makes sense to drop the `await` but we can still have the `instrument(info_span!`